step-release-vis/src/app/services/candidate_test.ts

```diff
@@ -41,4 +40,57 @@ describe('CandidateService', () => {
       expect(service.cands.get('2').polygons).toEqual(inputPolygons);
     });
   });
+
+  describe('#polygonHovered', () => {
+    it('test', () => {
+      service = new CandidateService();
```

Comment:
You could move this to a parent describe's setup/beforeEach as it is the same for almost all the tests, so you could reduce the actual testcode.

Reply:
done in new commit :)